server/game/cards/03_TWI/events/PoliticalPressure.ts

```diff
@@ -0,0 +1,28 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { RelativePlayer, TargetMode } from '../../../core/Constants';
+
+export default class PoliticalPressure extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3357486161',
+            internalName: 'political-pressure',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Discard a random card from your hand',
+            targetResolver: {
+                choosingPlayer: RelativePlayer.Opponent,
+                mode: TargetMode.Select,
+                choices: () => ({
```

Comment:
There's a little bit of nuance to this but the short version is that this implementation needs to use the `ifYouDoNot` property instead of being set up as a choice like this. As it is right now, the opponent could choose random discard when they have an empty hand and avoid creating the tokens. But the way "if you do not" works, if their hand is empty they _must_ accept the battle droid tokens. We'll need to add a test case for that.

The `ifYouDoNot` property should handle that correctly, though it's fairly new so it might be buggy. Lmk if you have any problems with it.